community/milind/Zarr_file_example/Zarr_file_example.py

```diff
@@ -31,22 +39,27 @@ def udf(bbox: fused.types.Bbox = None, layer: str = "ndvi", time: int = 2, targe
     else:
         print("No time dimension found in the dataset.")
 
-    # Selecting subset and tile for the current tile
-    buffer = ds.longitude[1]-ds.longitude[0]
-    ds_buffer = ds.sel(latitude=slice(maxy+buffer, miny-buffer), longitude=slice(minx-buffer, maxx+buffer)).isel(time=time)    
-    da=utils.da_fit_to_resolution(ds_buffer[layer], target_shape)
+    # Calculate buffer size (ensure float)
+    buffer = float(ds.longitude[1].values) - float(ds.longitude[0].values)
+    
+    # Select data with buffer
+    ds_buffer = ds.sel(
+        latitude=slice(float(maxy+buffer), float(miny-buffer)), 
+        longitude=slice(float(minx-buffer), float(maxx+buffer))
+    ).isel(time=time)
+    
+    # Process data
+    da = utils.da_fit_to_resolution(ds_buffer[layer], target_shape)
     da = da.sel(latitude=slice(miny, maxy), longitude=slice(minx, maxx))
     
-    # Reprojecting to Web Mercator for visualization
+    # Reproject
```

Comment:
Very much a nitpick, but I would keep this longer comment because I find it useful to explain why we are actually reprojecting to Web Mercator

```suggestion
    # Reproject to Web Mercator for visualization
```

(I fixed this UDF a while ago and it was doing a wrong reprojection resulting in slightly shifted visualization)